pages/pil/pdf/views/pil.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { Fragment } from 'react';
+import StandardConditions from '../../../common/views/pdf/standard-conditions';
+import ReactMarkdown from 'react-markdown';
+
+const PIL = ({ pil, content }) => {
+  const licenceHolder = pil.licenceHolder;
+  const establishment = pil.establishment;
+
+  const hasSpecies = pil.species && pil.species.length > 0;
+  const hasProcedures = pil.procedures && pil.procedures.length > 0;
+  const hasAdditionalConditions = pil.conditions && pil.conditions.length > 0;
+
+  return (
+    <Fragment>
+      <div className="logo"></div>
+
+      <h3 className="licence">PERSONAL LICENCE</h3>
+
+      <h1 className="licence-holder">{`${licenceHolder.firstName} ${licenceHolder.lastName}`}</h1>
+
+      <ReactMarkdown className="legal-preamble">{content.legalPreamble}</ReactMarkdown>
+
+      <section className="primary-establishment break">
+        <h2>Primary establishment</h2>
+        <ReactMarkdown>{establishment.address}</ReactMarkdown>
```

Comment:
This might need to go.

If it doesn't then it's super weird to only have the address and not the actual name of the establishment.

Reply:
Sounds like we need to pass in the `correctEstablishment` prop here too and hide the primary establishment details if it's not the one requesting the document.

Reply:
Yeah we want the establishment name in there as well, I think I missed it cause "University of Croydon" is the address in our seeds.